packages/legacy/src/Providers/components/AddEditProviderModal/AddEditProviderModal.tsx

```diff
@@ -533,18 +547,8 @@ export const AddEditProviderModal: React.FunctionComponent<IAddEditProviderModal
                     />
                   </FormGroup>
                 ) : null}
-                {fields?.insecure? (
-                  <Checkbox
-                    label="Skip certificate validation (if checked, the CA certificate won't be validated)"
-                    aria-label="Insecure connection checkbox"
-                    id="insecure-check"
-                    isChecked={fields.insecure.value}
-                    onChange={() =>
-                      fields.insecure?.setValue(!fields.insecure.value)
-                    }
-                  />
-                ) : null}
-                {fields?.insecure && !fields?.insecure?.value && fields?.caCertIfSecure && fields?.caCertFilenameIfSecure ? (
+
+                {fields?.insecureSkipVerify && !fields?.insecureSkipVerify?.value && fields?.caCertIfSecure && fields?.caCertFilenameIfSecure ? (
```

Comment:
with vmware we will need to show the cert even on insecure skip verify , because of an implementation detail of how we get the vddk image. so in the case of vmware we will need to show the ca cert even if user checks the "please don't check the cert" checkbox.

Reply:
That's the openstack cert field. For vmware, only the fingerprint is ever stored, and it is always displayed.